modules/gateways/payfast.php

```diff
@@ -154,7 +154,7 @@ function Payfast_nolocalcc(): void
  * @return array|string
  * @see https://developers.whmcs.com/payment-gateways/remote-input-gateway/
  */
-function Payfast_remoteinput(): array|string
+function Payfast_remoteinput(): array
```

Comment:
This will fail, since the function expects to return an array, but actually returns a string

Reply:
Thanks for that, I did a vice versa correction. Return is supposed to be string and not array